docs/DeveloperGuide.md

```diff
@@ -320,7 +320,24 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC01 - Search for person(s)**
+
+**MSS**
+
+1.  User searches for a person by name
+2.  AddressBook shows a list of persons which fit the criteria
+
+    Use case ends.
```

Comment:
![image](https://github.com/user-attachments/assets/dd23edf0-9870-430c-b070-3b8f19b43d16)

The formatting seems a little different from the examples given. Did they mention anything about it or is it ok

Reply:
I followed the formatting present from the sample use case in the sample DG, so I think should be fine

Reply:
Okay sounds good 🥇